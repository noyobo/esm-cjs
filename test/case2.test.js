const code = `import {
  root,
  templateMethods
} from "../../../../chunk-STXT7BRD.js";
import "../../../../chunk-UN2MHHJ5.js";
export {
  root as default,
  templateMethods
};`;

const transform = require('../lib/transform');

describe('transform import case1', () => {
  it('should work', () => {
    expect(transform(code)).toMatchSnapshot();
  });
});
