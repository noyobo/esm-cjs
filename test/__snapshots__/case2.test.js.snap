// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`transform import case1 should work 1`] = `
"const _chunkSTXT7BRDJs = require('../../../../chunk-STXT7BRD.js');

const root = _chunkSTXT7BRDJs.root;
const templateMethods = _chunkSTXT7BRDJs.templateMethods;

require('../../../../chunk-UN2MHHJ5.js');

exports.default = root;
exports.templateMethods = templateMethods;"
`;
